service/src/main/resources/db/migration/V272__message_thread_participant.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE message_thread_participant
+(
+    id                      uuid PRIMARY KEY         NOT NULL DEFAULT ext.uuid_generate_v1mc(),
+    created                 timestamp with time zone NOT NULL DEFAULT now(),
+    updated                 timestamp with time zone NOT NULL DEFAULT now(),
+    thread_id               uuid                     NOT NULL REFERENCES message_thread (id) ON DELETE CASCADE,
+    participant_id          uuid                     NOT NULL REFERENCES message_account (id) ON DELETE CASCADE,
+    last_message_timestamp  timestamp with time zone NOT NULL,  -- timestamp of last sent or received message
+    last_received_timestamp timestamp with time zone,           -- timestamp of last received message
+    last_sent_timestamp     timestamp with time zone,           -- timestamp of last sent message
```

Comment:
`last_sent_timestamp` is currently used, but might be used in the future. I thought it's better to compute it in this data migration rather than have more slow data migrations in the future.